espanol/torre/recepcion/recepcion.md

```diff
@@ -0,0 +1 @@
+Llegando a cierta distancia de la compañia que se acercaba, el general del ejército pudo darse cuenta de que eran embajadores que venían en paz.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.